core/src/main/java/tachyon/Users.java

```diff
@@ -42,11 +42,12 @@
   private final String mUserUnderFSFolder;
   /** Map from UserId to {@link tachyon.UserInfo} object **/
   private final Map<Long, UserInfo> mUsers;
+  private final TachyonConf mTachyonConf;
 
-  public Users(final String userUfsFolder) {
-
+  public Users( String userUfsFolder, TachyonConf tachyonConf) {
```

Comment:
@hsaputra Could you please also update the style check file to catch this? Thanks.